.github/CODEOWNERS

```diff
@@ -12,22 +12,22 @@
 /sdk/boms/azure-sdk-bom/                            @alzimmermsft @jonathangiles @srnagar @hemanttanwar @anuchandy
 
 # Service teams
-/sdk/appconfiguration/                              @mssfang @alzimmermsft @conniey @sima-zhu
-/sdk/batch/                                         @xingwu1 @bgklein @matthchr
-/sdk/core/                                          @alzimmermsft @jianghaolu @srnagar @hemanttanwar @anuchandy
-/sdk/core/azure-core-tracing-opentelemetry/         @samvaity @alzimmermsft
-/sdk/cosmos/                                        @moderakh @kushagraThapar @David-Noble-at-work @kirankumarkolli @mbhaskar
+/sdk/appconfiguration/                              @mssfang @alzimmermsft @conniey @sima-zhu                     %AzConfig
+/sdk/batch/                                         @xingwu1 @bgklein @matthchr                                   %Batch
+/sdk/core/                                          @alzimmermsft @jianghaolu @srnagar @hemanttanwar @anuchandy   %Azure.Core
+/sdk/core/azure-core-tracing-opentelemetry/         @samvaity @alzimmermsft 
```

Comment:
Should we add a tag for tracing?

Reply:
Your call 🙂. What would help more?

Reply:
Tracing and core don't always go hand in hand for Java, I think it would make sense to add a tag for tracing specifically.

Reply:
Is there a label for tracing already?

Reply:
I think we used `Monitor` label, but don't know if that would still be the correct one.
cc: @joshfree

Reply:
For azure-core-tracing we've converged on using azure-core for all the languages.

Monitor is going to be used for Monitor Exporter and Monitor Track 2 SDKs

Reply:
Thanks @joshfree! I'll go ahead and update this.